### PR TITLE
Add get /forecast endpoint

### PR DESCRIPTION
**Pull request by tylorschafer**
- Added forecast router file 
- Added functions for fetch from Google Geocode and Darksky 
- Get /forecast endpoint fully functional.

### CODE REVIEW

**Comment by tylorschafer** [comment]
Looking for tips on how to refactor this now that its working.

File: routes/api/v1/forecasts.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  const daily = {}
+  daily.summary = darkResponse.daily.summary
+  daily.icon = darkResponse.daily.icon
+  daily.data = []
+  darkResponse.daily.data.slice(0, 7).forEach(function (item, index) {
+    const day = {}
+    day.time = item.time
+    day.summary = item.summary
+    day.icon = item.icon
+    day.sunriseTime = item.sunriseTime
```
Comment (evettetelyas) [inline routes/api/v1/forecasts.js:77]: I think here you can invoke a formatter helper and have all these long format functions in there!

Comment (tylorschafer) [reply routes/api/v1/forecasts.js:77]: I was just about to start working on that, thanks!

File: routes/api/v1/forecasts.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
+var express = require('express')
+var router = express.Router()
+
+const environment = process.env.NODE_ENV || 'development'
+const configuration = require('../../../knexfile')[environment]
+const database = require('knex')(configuration)
+const fetch = require('node-fetch')
+
+function findByKey (key) {
+  return database.select('id').from('users').where('api_key', key).first()
```
Comment (evettetelyas) [inline routes/api/v1/forecasts.js:10]: you can move this into a User model function since youll prob use it in other routes

Comment (tylorschafer) [reply routes/api/v1/forecasts.js:10]: I ended up moving that function because I did have to use it again later. Thanks for the input!

**Pull request merged**